pkg/reconciler/accessor/core/secret_test.go

```diff
@@ -132,16 +129,14 @@ func TestReconcileSecretCreate(t *testing.T) {
 
 func TestReconcileSecretUpdate(t *testing.T) {
 	ctx, cancel, _ := SetupFakeContextWithCancel(t)
-	grp := errgroup.Group{}
+	waitInformers := func() {}
```

Comment:
I'd move defer down after line 140 and get rid of this definition of `waitInformers`.
If we're going to panic inbetween -- I think it's fine if we leak the context at that point.
Same elsewhere.

Reply:
Done.